docs/releases/v2020.2.rst

```diff
@@ -0,0 +1,178 @@
+Gluon 2020.2
+=============
+
+Added hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+* GL.iNet
+
+  - GL-AR750S
+
+* TP-Link
+
+  - CPE220 (v3)
+
+ipq40xx-generic
+~~~~~~~~~~~~~~~
+
+* EnGenius
+
+  - ENS620EXT [#outdoor]_
+
+* Linksys
+
+  - EA6350 (v3)
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+* Cudy
+
+  - WR1000
+
+lantiq-xway
+~~~~~~~~~~~
+
+* NETGEAR
+
+  - DGN3500B
+
+lantiq-xrx200
+~~~~~~~~~~~~~
+
+* TP-Link
+
+  - TD-W8970
+
+
+x86-legacy [#newtarget]_
+~~~~~~~~~~
+
+* Devices older than the Pentium 4
+
+
+.. [#newtarget]
+  This is a new target in this release
+
+
+.. [#oudoor]
+  This device is supposed to be setup outdoors and will therefore have it's outdoor mode flag automatically enabled.
+
+
+Major changes
+-------------
+
+Device Classes
+==============
+
+Devices not can be categorised into device classes. This device class can determine which features
```

Comment:
```suggestion
Devices now can be categorised into device classes. This device class can determine which features
```
typo

Reply:
Hmm, that's not quite accurate either, as it's already done. "Devices have been categorized into device classes."?

Reply:
Why not keep it simple: "Devices are now categorized into device classes."?